<launch>
  <node name="player" pkg="rosbag" type="play" output="screen" args="-l /home/oshri/yellow.bag"></node>
  <node name="drop" pkg="topic_tools" type="drop" output="screen" args="/ardrone/bottom/image_raw 1 2"></node>
  <node name="image_converter" pkg="ardrone_project" type="image_converter.py" output="screen"></node>
  <node name="end_detection" pkg="ardrone_project" type="end_detection.py" output="screen"></node>
</launch>
